db/migrate/20211001151212_update_patient_table_string_lengths.rb

```diff
@@ -0,0 +1,185 @@
+class UpdatePatientTableStringLengths < ActiveRecord::Migration[6.1]
+  def up
+    # Identification
+    change_column :patients, :first_name, :string, limit: 200
+    change_column :patients, :last_name, :string, limit: 200
+    change_column :patients, :middle_name, :string, limit: 200
+    change_column :patients, :sex, :string, limit: 16
```

Comment:
I have a few concerns about limiting the field lengths in this way. This introduces a dependency between the database and the allowed enumerated fields that may not be obvious to someone who is less familiar with the system. It feels relatively plausible that someone would change the enumerated values allowed for a string, and not realize that this means they also have to do a migration. 

Also the mapping between enumerated value length and the limits isn't explicitly defined anywhere, it's just defined by us looking at the allowed strings.

Overall, it just feels like doing this may be confusing for future developers. I'm still fine if we want to individually limit column lengths, but I do think if we are going to do that, we may as well go low as possible and tie in directly to the longest allowed enumerated value.

Reply:
yeah I did think about that and you have good points, how do we feel about just limiting these to 200 characters for now? we'll probably have to come up with another solution soon if we continue to add more columns, but this will buy us some time to do that

Reply:
Based on some slack convos, I think @Bialogs has some thoughts on this, so I'm interested to see what him and @tstrass think. It looks like there are 74 fields, so if we limit them all to 200 from 255, that buys us `(55 * 74) / 200 = 20` more string fields. Which I think is enough for our immediate needs.

Reply:
Good discussion. I would support the blanket 200 char limit for now, with the knowledge that it will not meet our needs for very long, and that our next move should be to offload set(s) of lesser used attributes to other table(s).

Reply:
this has been addressed